src/utils/firebase.js

```diff
@@ -74,3 +80,40 @@ export const download = (uuid) => {
     .once('value')
     .then((snapshot) => snapshot.val());
 };
+
+export const createLink = (path) => {
+  if (__DEV__) {
+    return Linking.makeUrl(path);
+  } else {
+    return fetch(`https://firebasedynamiclinks.googleapis.com/v1/shortLinks?key=${EXPO_FIREBASE_API_KEY}`, {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        dynamicLinkInfo: {
+          domainUriPrefix: EXPO_FIREBASE_DOMAIN_URI,
+          link: `${EXPO_FIREBASE_URL_PREFIX}/${path}`,
+          androidInfo: {
+            androidPackageName: 'com.discin.discin',
+            androidMinPackageVersionCode: '4',
```

Comment:
Should be 5 (next live version)

Reply:
Tu commentes tes propres PR :)

Reply:
Ouais, j'ai peur d'oublier de le remettre bien après avoir testé